app/test_main.py

```diff
@@ -1,3 +1,20 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize("cat_years, dog_years, expected", [
+    (0, 0, [0, 0]),
+    (13, 13, [0, 0]),
+    (14, 14, [0, 0]),
```

Comment:
The test case `(14, 14, [0, 0])` is repeated from the previous line `(13, 13, [0, 0])`. According to the checklist, you should avoid testing the same case multiple times. Consider removing one of these cases to avoid redundancy.